Gabriele/25-forLoops.js

```diff
@@ -0,0 +1,138 @@
+// 1. Addition.
+// Write a program to add up the numbers 1 to 20.
+
+for (let i = 1; i <= 20; i++) {
+    console.log(i);
+};
+
+
+// 2. There are i bottles of beer on the wall.
+// Write a program that will output, "There is one bottle of beer on the wall." "There are two bottles of beer on the wall" up until there are five bottles.
+
+
+let beerBot = 1;
+for (i = 1; i <= 5; i++) {
```

Comment:
we need if statement to manage
if one "There is one bottle of beer on the wall."
more than one  "There are two bottles of beer on the wall"